packages/contracts/contracts/L2/messaging/L2CrossDomainMessenger.sol

```diff
@@ -36,18 +32,18 @@ contract L2CrossDomainMessenger is
     address internal xDomainMsgSender = Lib_DefaultValues.DEFAULT_XDOMAIN_SENDER;
     address public l1CrossDomainMessenger;
 
+
     /***************
      * Constructor *
      ***************/
 
     constructor(
         address _l1CrossDomainMessenger
-    )
-        ReentrancyGuard()
```

Comment:
We no longer need `ReentrancyGuard` because `relayMessage` uses `msg.sender` to verify the incoming message instead of the `L1MessageSender` opcode.